gapic-common/lib/gapic/grpc/service_stub/rpc_call.rb

```diff
@@ -135,7 +135,7 @@ def call request, options: nil, format_response: nil, operation_callback: nil, s
             Thread.new { operation.execute }
           else
             response = operation.execute
-            response = format_response.call response if format_response
+            response = format_response.call response, operation if format_response
```

Comment:
I don't understand why `operation` is being added to the `format_response` callback when the `operation_callback` already exists. Shouldn't `operation_callback` be called for every paged request/response instead?

Reply:
This is because `format_response` is used to create the `PagedEnumerable`. See `gapic-generator/templates/default/service/client/method/def/_response.erb`.

It is a bit odd to include here, but I didn't see an obvious alternative and I don't think it hurts.